### ISSUE TITLE
Add Web Interface and/or REST API

### ISSUE THREAD

**Issue by mondayrain** [yask123/Instant-Music-Downloader#27]
Some of my not-so-technical friends have expressed interest in being able to use this tool, and I don't think I would be successful in trying to convince them to use the command line :D

Thinking of creating a very simple web interface and hosting it on a DO droplet somewhere, if that is cool with you @yask123. If I did, would you want the page to be called "Instant Music Downloader" or something else..?

**Comment by yask123** [comment]
I did make one Django application with REST API:
https://github.com/yask123/InstantMusicDownloader-WebApp 

Demo: https://www.youtube.com/watch?v=iWJcg8pIHMY
And hosted it on Free tier AWS servers (which had bandwidth limit of 15 GB).

People started downloading tons of songs and it nearly exceeded the free limit, and so I stopped the online server.

An alternate would be to create a user friendly desktop application?
I quickly ported this web app to desktop app using `electron`:

![tc8ylaw](https://cloud.githubusercontent.com/assets/1812021/10323069/6ad3ac82-6c9e-11e5-9e91-5708b05c6d4b.png)
